src/blog/2023/06/dashboard-announcement.md

```diff
@@ -0,0 +1,42 @@
+---
+title: The Next Step in Data Visualization - Announcing the Successor to the Node-RED Dashboard
+subtitle: FlowForge's Journey Towards a New Node-RED Dashboard
+description: This article unveils FlowForge's plans to build the official successor to the Node-RED Dashboard.
+date: 2023-06-16
+authors: ["joe-pavitt", "marian-demme"]
+# image: 
+tags:
+    - posts
+---
+
+For the past several years, the Node-RED Dashboard has been an indispensable tool for many Node-RED users. It has offered a seamless way to create live dashboards, enabling the quick and intuitive creation of user interfaces for Node-RED flows. However, as the saying goes, "all good things must come to an end." 
+<!--more-->
+We at FlowForge have identified a significant need for a modern, interactive data visualization and dashboard solution. In response, we've decided to embark on an exciting journey: the creation of the official successor to the Node-RED Dashboard.
+
+## The Problem at Hand
+
+The original Node-RED Dashboard is based on Angular v1, which is no longer maintained. Although small patches have been and will continue to be applied on a "best can do" basis, there will be no major feature upgrades. The lack of ongoing maintenance and updates has the potential to lead to underlying security breakages, a risk we are not comfortable taking. We have recognized the need to innovate and adapt, which is why we are creating a completely new project to replace the existing Node-RED Dashboard.
+
+## The Solution
+
+The successor to the Node-RED Dashboard will be a completely new project, published as a separate package. This new project will take the reins from the old dashboard and guide us into the future. However, the Node-RED community can rest assured that the project will stay under the Apache 2.0 licence. This ensures that the core principles of open-source and community-driven development will remain intact.
+
+In line with our open core approach, we will provide vendors the opportunity to build premium features around the core. This allows businesses to customize and enhance their experience with the new dashboard, while still maintaining the integrity of the open-source project.
+
+At FlowForge, we are not just creating a new dashboard, but also adding an Enterprise Extension with additional functionalities. This extension will provide features and capabilities geared towards the needs of larger organizations and enterprises.
```

Comment:
Unless we have an example of what these will be I would remove this sentence too. It will just create questions that we can't or don't want to answer.

Reply:
[Remove Enterprise Extension part](https://github.com/flowforge/website/pull/847/commits/f10a0a3853edf5acdee31a4ea40e8c25d5133b50)